aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -395,7 +396,8 @@ Tensor norm_sparse(const SparseTensor& self, const optional<Scalar>& p, IntArray
   }
   TORCH_CHECK(keepdim == false, "norm_sparse currently does not support keepdim=True");
   TORCH_CHECK(!dtype.has_value(), "norm_sparse currently does not support 'dtype' argument");
-  auto p_ = p.value_or(2.0);
+  constexpr auto TWO = 2.0;
+  auto p_ = p.value_or(TWO);
```

Comment:
is this just because of some clang-tidy check for magic numbers?

Reply:
Yes. Although, frankly it feels like a magic number here

Reply:
lol yeah I mean replacing `2.0` with `TWO` doesn't really make it any less magic in my mind